lsp-dart-utils.el

```diff
@@ -41,13 +41,20 @@ FLUTTER_ROOT environment variable and the PATH environment variable."
   :risky t
   :type '(choice directory nil))
 
+(defcustom lsp-dart-flutter-executable "flutter"
```

Comment:
Could you add this variable to the changelog file and to the README settings section please?

Reply:
Should I create a new version `1.17.6` in the changelog file for this?

Reply:
Yes, please :)

Reply:
I'll generate the 1.17.6 release after the merge